[2024-04-19 10:58:30,566: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 10:58:30,571: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 10:58:30,575: INFO: common: created directory at: artifacts]
[2024-04-19 10:58:30,577: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 10:58:32,127: INFO: 1434958058: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 205A:20CAE:786833:96754C:6621E424
Accept-Ranges: bytes
Date: Fri, 19 Apr 2024 03:58:25 GMT
Via: 1.1 varnish
X-Served-By: cache-qpg1255-QPG
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1713499105.311967,VS0,VE276
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 26e3d46566fcbebfa1dbf50b513037e6e082343c
Expires: Fri, 19 Apr 2024 04:03:25 GMT
Source-Age: 0

]
[2024-04-19 11:46:55,189: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 11:46:55,193: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 11:46:55,195: INFO: common: created directory at: artifacts]
[2024-04-19 11:46:55,196: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 11:46:55,197: INFO: 1434958058: File already exists of size: ~ 7718 KB]
[2024-04-19 12:05:16,542: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 12:06:31,576: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 13:47:20,470: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:47:20,483: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:47:20,484: INFO: common: created directory at: artifacts]
[2024-04-19 13:47:20,484: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 13:47:23,163: INFO: 3489392330: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4662:8B287:A5F03E:C3E403:6622136F
Accept-Ranges: bytes
Date: Fri, 19 Apr 2024 06:47:15 GMT
Via: 1.1 varnish
X-Served-By: cache-tyo11949-TYO
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1713509235.430303,VS0,VE397
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 3a629cc7db0c933d6ea48392fcc717a35708ac37
Expires: Fri, 19 Apr 2024 06:52:15 GMT
Source-Age: 0

]
[2024-04-19 13:51:29,603: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-19 13:51:29,609: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:51:29,609: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:51:29,609: INFO: common: created directory at: artifacts]
[2024-04-19 13:51:29,612: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 13:51:31,760: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4662:8B287:A5F03E:C3E403:6622136F
Accept-Ranges: bytes
Date: Fri, 19 Apr 2024 06:51:24 GMT
Via: 1.1 varnish
X-Served-By: cache-tyo11931-TYO
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1713509485.633653,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: bf118507d052c4bd75d62a0db7a91538d9e8fec3
Expires: Fri, 19 Apr 2024 06:56:24 GMT
Source-Age: 249

]
[2024-04-19 13:51:31,908: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-04-19 13:54:55,346: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-19 13:54:55,346: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:54:55,346: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:54:55,351: INFO: common: created directory at: artifacts]
[2024-04-19 13:54:55,351: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 13:54:55,352: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-04-19 13:54:55,473: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-04-19 13:54:56,346: INFO: stage_02_data_validation: *******************]
[2024-04-19 13:54:56,346: INFO: stage_02_data_validation: >>>>>> stage Data Validation stage started <<<<<<]
[2024-04-19 13:54:56,352: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:54:56,352: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:54:56,352: INFO: common: created directory at: artifacts]
[2024-04-19 13:54:56,352: INFO: common: created directory at: artifacts/data_validation]
[2024-04-19 13:54:56,357: INFO: stage_02_data_validation: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-04-19 13:55:02,672: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 13:55:03,133: INFO: stage_03_data_transformation: *******************]
[2024-04-19 13:55:03,133: INFO: stage_03_data_transformation: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-04-19 13:55:03,142: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:55:03,145: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:55:03,145: INFO: common: created directory at: artifacts]
[2024-04-19 13:55:03,145: INFO: common: created directory at: artifacts/data_transformation]
[2024-04-19 13:55:07,973: INFO: stage_03_data_transformation: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-04-19 13:56:26,542: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 13:56:26,792: INFO: stage_04_model_trainer: *******************]
[2024-04-19 13:56:26,792: INFO: stage_04_model_trainer: >>>>>> stage Model Training stage started <<<<<<]
[2024-04-19 13:56:26,792: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:56:26,792: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:56:26,792: INFO: common: created directory at: artifacts]
[2024-04-19 13:56:26,792: INFO: common: created directory at: artifacts/model_trainer]
[2024-04-19 13:56:38,332: ERROR: stage_04_model_trainer: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`]
Traceback (most recent call last):
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 23, in <module>
    obj.main()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 16, in main
    model_trainer_config.train()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\components\model_trainer.py", line 34, in train
    trainer_args = TrainingArguments(
  File "<string>", line 124, in __init__
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1551, in __post_init__
    and (self.device.type != "cuda")
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 2027, in device
    return self._setup_devices
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\utils\generic.py", line 63, in __get__
    cached = self.fget(obj)
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1937, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`
[2024-04-19 13:57:59,562: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 13:57:59,812: INFO: stage_04_model_trainer: *******************]
[2024-04-19 13:57:59,819: INFO: stage_04_model_trainer: >>>>>> stage Model Training stage started <<<<<<]
[2024-04-19 13:57:59,822: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 13:57:59,822: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 13:57:59,822: INFO: common: created directory at: artifacts]
[2024-04-19 13:57:59,822: INFO: common: created directory at: artifacts/model_trainer]
[2024-04-19 13:58:06,692: ERROR: stage_04_model_trainer: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`]
Traceback (most recent call last):
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 23, in <module>
    obj.main()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 16, in main
    model_trainer_config.train()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\components\model_trainer.py", line 34, in train
    trainer_args = TrainingArguments(
  File "<string>", line 124, in __init__
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1551, in __post_init__
    and (self.device.type != "cuda")
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 2027, in device
    return self._setup_devices
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\utils\generic.py", line 63, in __get__
    cached = self.fget(obj)
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1937, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`
[2024-04-19 14:15:22,051: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 14:15:22,331: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:15:22,331: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:15:22,335: INFO: common: created directory at: artifacts]
[2024-04-19 14:15:22,338: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:16:32,491: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:16:32,499: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:16:32,500: INFO: common: created directory at: artifacts]
[2024-04-19 14:16:32,500: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:21:01,846: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 14:21:02,146: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:21:02,146: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:21:02,154: INFO: common: created directory at: artifacts]
[2024-04-19 14:21:02,156: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:23:20,244: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 14:23:20,498: INFO: stage_04_model_trainer: *******************]
[2024-04-19 14:23:20,498: INFO: stage_04_model_trainer: >>>>>> stage Model Training stage started <<<<<<]
[2024-04-19 14:23:20,501: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:23:20,501: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:23:20,501: INFO: common: created directory at: artifacts]
[2024-04-19 14:23:20,501: INFO: common: created directory at: artifacts/model_trainer]
[2024-04-19 14:23:27,532: ERROR: stage_04_model_trainer: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`]
Traceback (most recent call last):
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 23, in <module>
    obj.main()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 16, in main
    model_trainer_config.train()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\components\model_trainer.py", line 34, in train
    trainer_args = TrainingArguments(
  File "<string>", line 124, in __init__
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1551, in __post_init__
    and (self.device.type != "cuda")
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 2027, in device
    return self._setup_devices
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\utils\generic.py", line 63, in __get__
    cached = self.fget(obj)
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\transformers\training_args.py", line 1937, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`
[2024-04-19 14:26:24,400: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 14:26:24,696: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:26:24,696: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:26:24,707: INFO: common: created directory at: artifacts]
[2024-04-19 14:26:24,707: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:29:37,634: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:29:37,634: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:29:37,634: INFO: common: created directory at: artifacts]
[2024-04-19 14:29:37,634: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:29:58,773: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 14:29:59,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:29:59,097: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:29:59,097: INFO: common: created directory at: artifacts]
[2024-04-19 14:29:59,097: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:37:27,492: INFO: rouge_scorer: Using default tokenizer.]
[2024-04-19 14:48:16,339: INFO: server: Started server process [3788]]
[2024-04-19 14:48:16,339: INFO: on: Waiting for application startup.]
[2024-04-19 14:48:16,348: INFO: on: Application startup complete.]
[2024-04-19 14:48:16,349: INFO: server: Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)]
[2024-04-19 14:51:25,617: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 14:51:25,620: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 14:51:25,620: INFO: common: created directory at: artifacts]
[2024-04-19 14:51:25,620: INFO: common: created directory at: artifacts/model_evaluation]
[2024-04-19 14:57:54,101: INFO: server: Shutting down]
[2024-04-19 14:57:54,215: INFO: on: Waiting for application shutdown.]
[2024-04-19 14:57:54,217: INFO: on: Application shutdown complete.]
[2024-04-19 14:57:54,218: INFO: server: Finished server process [3788]]
[2024-04-19 15:00:46,280: INFO: server: Started server process [17752]]
[2024-04-19 15:00:46,280: INFO: on: Waiting for application startup.]
[2024-04-19 15:00:46,280: INFO: on: Application startup complete.]
[2024-04-19 15:00:46,280: INFO: server: Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)]
[2024-04-19 15:02:30,083: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 15:02:47,479: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 15:03:59,435: INFO: config: PyTorch version 2.2.2 available.]
[2024-04-19 15:03:59,836: INFO: main: >>>>>> stage Data Ingestion Code started <<<<<<]
[2024-04-19 15:03:59,839: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 15:03:59,841: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 15:03:59,841: INFO: common: created directory at: artifacts]
[2024-04-19 15:03:59,842: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-19 15:03:59,842: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-04-19 15:03:59,982: INFO: main: >>>>>> stage Data Ingestion Code completed <<<<<<]
[2024-04-19 15:03:59,982: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-04-19 15:03:59,985: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 15:03:59,986: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 15:03:59,987: INFO: common: created directory at: artifacts]
[2024-04-19 15:03:59,987: INFO: common: created directory at: artifacts/data_validation]
[2024-04-19 15:03:59,989: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-04-19 15:03:59,989: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-04-19 15:03:59,993: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-19 15:03:59,995: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-19 15:03:59,995: INFO: common: created directory at: artifacts]
[2024-04-19 15:03:59,996: INFO: common: created directory at: artifacts/data_transformation]
[2024-04-19 15:04:05,219: ERROR: main: [Errno 22] Invalid argument: 'C:/Users/Daro/OneDrive/Documents/Deep-Learning-Project/NLP-Text_summarization/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\main.py", line 33, in <module>
    data_transformation.main()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\pipeline\stage_03_data_transformation.py", line 14, in main
    data_transformation.convert()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\src\textSummarizer\components\data_transformation.py", line 32, in convert
    dataset_samsum_pt.save_to_disk(os.path.join(self.config.root_dir,"samsum_dataset"))
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\datasets\dataset_dict.py", line 1297, in save_to_disk
    dataset.save_to_disk(
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\datasets\arrow_dataset.py", line 1536, in save_to_disk
    for job_id, done, content in Dataset._save_to_disk_single(**kwargs):
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\datasets\arrow_dataset.py", line 1560, in _save_to_disk_single
    writer = ArrowWriter(
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\datasets\arrow_writer.py", line 337, in __init__
    self.stream = self._fs.open(fs_token_paths[2][0], "wb")
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\fsspec\spec.py", line 1293, in open
    f = self._open(
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\fsspec\implementations\local.py", line 184, in _open
    return LocalFileOpener(path, mode, fs=self, **kwargs)
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\fsspec\implementations\local.py", line 306, in __init__
    self._open()
  File "C:\Users\Daro\OneDrive\Documents\Deep-Learning-Project\NLP-Text_summarization\vtextSummarization\lib\site-packages\fsspec\implementations\local.py", line 311, in _open
    self.f = open(self.path, mode=self.mode)
OSError: [Errno 22] Invalid argument: 'C:/Users/Daro/OneDrive/Documents/Deep-Learning-Project/NLP-Text_summarization/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow'
[2024-04-19 15:04:43,995: INFO: server: Shutting down]
[2024-04-19 15:04:44,111: INFO: on: Waiting for application shutdown.]
[2024-04-19 15:04:44,112: INFO: on: Application shutdown complete.]
[2024-04-19 15:04:44,112: INFO: server: Finished server process [17752]]
